cmd/restic/cmd_backup.go

```diff
@@ -60,7 +60,7 @@ given as the arguments.
 		term := termstatus.New(globalOptions.stdout, globalOptions.stderr, globalOptions.Quiet)
 		t.Go(func() error { term.Run(t.Context(globalOptions.ctx)); return nil })
 
-		err := runBackup(backupOptions, globalOptions, term, args)
+		err := runBackup(backupOptions, globalOptions, term, args, globalOptions.Quiet)
```

Comment:
It seems to me that an additional parameter is unnecessary, and the value can be read from other parameters that are passed to the function already (```runBackup((...), globalOptions, (...), globalOptions.Quiet)```).

Reply:
see above